fitsio/test.py

```diff
@@ -1316,6 +1316,10 @@ def testTableWriteBadString(self):
 
                     except ValueError:
                         got_error=True
+                    except TypeError:
```

Comment:
Can we scope this error to only coming from numpy?

Reply:
It doesn't look to be subclassed.  do you have something else in mind?

Reply:
Maybe test against the exact error string from numpy?

Reply:
Or a substring of it?

```python
except TypeError as e:
    if str(e) == 'Empty data-type':
         got_error = True
    else:
        raise
```

Reply:
Not sure if raise is right for the else clause